contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -97,30 +106,45 @@ func main() {
 	cmd := commands.NewCommand(
 		commands.Metadata{
 			Name:       "gnodev",
-			ShortUsage: "gnodev [flags] [path ...]",
-			ShortHelp:  "runs an in-memory node and gno.land web server for development purposes.",
+			ShortUsage: "gnodev [flags] ",
```

Comment:
```suggestion
			ShortUsage: "gnodev [flags]",
```

what if we had instead a syntax like `gnodev [<local|staging>]`, and the default was on `local`, and we don't keep flags on the top level

Reply:
hmm we could do that, let me try

Reply:
I tried it out, and from a file structure perspective, it's cool. However, having both commands, `gnodev` and `gnodev local`, perform the same function is a bit confusing. I could move all the flags into the local command to keep the root command clean, but that would also create confusion, as you would need to check `gnodev local` to see the correct list of flags for the `local` command.

You can check a preview here: https://github.com/gfanton/gno/tree/preview/feat/lazy-loader/local-command/contribs/gnodev/cmd/gnodev

The main changes are `command_local.go`, `app_config.go`, and `main.go` (`main.go` is not definitive, as I wasn't sure how to handle `default` behaviors). 

I really want to keep that file structure as it file way cleaner anyway, just not sure to keep this `local` subcommand.

wdyt ?

Reply:
Ok, I've pushed an update to make `gnodev` and `gnodev local` act the same. I feel it's a bit cleaner but can be annoying over time. We can experiment and keep it that way; if we feel it's too annoying, we can roll back.  
By the way, I had to update the command to make this work. It was not an easy task due to the way the command is implemented.

Reply:
I think it looks good